src/lib/common/data_getter.ts

```diff
@@ -0,0 +1,18 @@
+import type { FeatureCollection } from "geojson";
+import authoritiesUrl from "../../../assets/authorities.geojson?url";
+
+export async function getAuthoritiesData() {
+  const resp = await fetch(authoritiesUrl);
+  const body = await resp.text();
+  const json: FeatureCollection = JSON.parse(body);
+  return Promise.resolve(json);
```

Comment:
Just `return json;` should be fine. "If the return value of an async function is not explicitly a promise, it will be implicitly wrapped in a promise." from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function